index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+<var>conforming document</var> by performing the following steps:
+            <ol class="algorithm">
+              <li>
+Set <var>securingMechanismVerified</var> to the result of executing the
+<a href="#verify-securing-mechanism">securing mechanism verification
+algorithm</a> with <var>inputBytes</var> and <var>mediaType</var> passed as
+inputs to the algorithm.
+              </li>
+              <li>
+Set <var>status</var>, <var>document</var>, <var>mediaType</var>,
+<var>warnings</var>, and <var>errors</var> in <var>verified</var> to their
+respective properties in <var>securingMechanismVerified</var>.
+              </li>
+              <li>
+If <var>verified</var>.<var>status</var> is `false`, the provided
+<var>inputBytes</var> contains a document that cannot be considered to be
+secured. Return <var>verified</var>, which will contain warning and error
+information related to the verification failure.
+              </li>
+            </ol>
+          </li>
+          <li>
+Ensure that the <var>verified</var>.<var>document</var> is a
+<a>conforming document</a> by performing the following steps:
+            <ol class="algorithm">
+              <li>
+Set <var>dataModelVerified</var> to the result of executing the
+<a href="#verify-data-model">data model verification algorithm</a>
+with <var>document</var> passed as input to the algorithm.
+              </li>
+              <li>
+Set <var>status</var>, <var>document</var>, <var>warnings</var>, and
+<var>errors</var> in <var>verified</var> to their respective properties in
+<var>dataModelVerified</var>.
+              </li>
+              <li>
+If <var>verified</var>.<var>status</var> is `false`, the provided
+<var>inputBytes</var> contains a non-conforming document. Return
+<var>verified</var>, which will contain warning and error information
+related to the verification failure.
+              </li>
+            </ol>
+          </li>
+          <li>
+Return <var>verified</var>.
+          </li>
+        </ol>
+
+        <p>
+The steps for verifying the state of the securing mechanism and verifying
+that the input document is a <a>conforming document</a> MAY be performed in
+a different order than the one provided above as long as the
+implementation returns errors for the same inputs.
+Implementations MAY produce different errors than described above.
+        </p>
+
+      </section>
+
+      <section class="normative">
+        <h3>Verify Data Model</h3>
+
+        <p>
+The algorithm in this section can be used to determine if a provided document is
+conformant to the data model provided in this specification. This algorithm
+takes a document (object <var>inputDocument</var>) and produces an object
+(<var>dataModelVerified</var>) that contains the following:
+        </p>
+
+        <ul>
+          <li>
+a verification status (boolean <var>status</var>)
+          </li>
+          <li>
+a verified document (object <var>document</var>)
```

Comment:
```suggestion
a validated document (object <var>document</var>)
```

Reply:
Fixed in 20f9b7423afe930f67a2a8baa5d9a5627f74f4f0.

Reply:
Thanks, resolving.